cartographer/parsers/jsonapi_parser.py

```diff
@@ -49,7 +49,7 @@ def attributes(self):
         return self.json_data.get("attributes", {})
 
     def attribute(self, name):
-        return self.attributes().get(name)
+        return self.attributes().get(name, None)
```

Comment:
Good question. I grepped for all usages of this function and found none are surrounded by `try`/`except` but all of them do some sort of check before hand to make sure the attribute is there.